src/adapters/output/index.ts

```diff
@@ -3,19 +3,20 @@ import { DataSource } from 'typeorm';
 import { DATABASE, DATABASE_HOST, DATABASE_PASSWORD, DATABASE_PORT, DATABASE_USER } from '@/env';
 import { Customer } from '@/domain/entities/customer';
 import { Product } from '@/domain/entities/product';
+import { Order } from "@/domain/entities/order";
 
 export const AppDataSource = new DataSource({
-  type: 'postgres',
-  host: DATABASE_HOST,
-  port: DATABASE_PORT,
-  username: DATABASE_USER,
-  password: DATABASE_PASSWORD,
-  database: DATABASE,
-  entities: [Customer, Product],
-  synchronize: true,
-  logging: ['log', 'error'],
-  // subscribers: [],
-  migrations: [],
+    type: 'postgres',
+    host: DATABASE_HOST,
+    port: DATABASE_PORT,
+    username: DATABASE_USER,
+    password: DATABASE_PASSWORD,
+    database: DATABASE,
+    entities: [Customer, Product, Order],
+    synchronize: true,
+    logging: ['log', 'error'],
+    // subscribers: [],
```

Comment:
These two lines commented  we can remove?

Reply:
I'll remove it